### PR TITLE
Set up package for TikTok integration

### PR DESCRIPTION
**Pull request by nickuya**
analytics.js allows us to build 3rd party pixel integrations into the main library by adding packages in `lib/integrations.js`

This PR creates a new package for the TikTok pixel in a `integrations` subdirectory - mostly just setup for now.

To come in later PRs:
- Additional implementation of the TikTok Pixel
- Integration with analytics.js

### CODE REVIEW

File: integrations/analytics.js-integration-tiktok-pixel/lib/index.ts [hunk 1]
```diff
@@ -0,0 +1,23 @@
+import Integration from '@segment/analytics.js-integration';
+
+const TIKTOK_PIXEL_GLOBAL = 'ttq';
```
Comment (jasonmiraglia) [inline integrations/analytics.js-integration-tiktok-pixel/lib/index.ts]: Just curious of the significance of the `q` in this? I would have expected `ttp` or `ttg` or `ttpg`

**Review by jasonmiraglia — APPROVED** [review]

File: integrations/analytics.js-integration-tiktok-pixel/lib/index.ts (hunk 1 above)
Comment (nickuya) [reply integrations/analytics.js-integration-tiktok-pixel/lib/index.ts]: I actually have no idea. That's how they name the global variable to store this object in all of their documentation.

Facebook's pixel uses `fbq`, Twitter ... you guessed it - `twq` - but couldn't not find any information about the significance of `q` at all!

Comment (nickuya) [reply integrations/analytics.js-integration-tiktok-pixel/lib/index.ts]: Well, Googling was no help - but ChatGPT to the rescue.

> Tracking pixels often have the letter 'q' at the end of their variable names as a convention to indicate that the variable is a queue or a list of tracking pixels to be loaded.

Comment (jasonmiraglia) [reply integrations/analytics.js-integration-tiktok-pixel/lib/index.ts]: Oh! Nice work ChatGPT

**Pull request merged**